airbyte-webapp-e2e-tests/cypress/integration/source.spec.ts

```diff
@@ -32,3 +42,60 @@ describe("Source main actions", () => {
     cy.get("div").contains(sourceName).should("not.exist");
   });
 });
+
+describe("Unsaved changes modal", () => {
+  beforeEach(() => {
+    initialSetupCompleted();
+  });
+
+  it("Check leaving Source page without any changes", () => {
+    goToSourcePage();
+    openNewSourceForm();
+
+    openHomepage();
+
+    cy.url().should("include", `/onboarding`);
```

Comment:
nit: use the same style of quotes throughout, unless you're using the interpolation feature of the backtick strings